package-lock.json

```diff
@@ -8213,6 +8213,12 @@
         "url": "https://github.com/sponsors/sindresorhus"
       }
     },
+    "node_modules/openapi-types": {
```

Comment:
why did you make changes in package-lock?

Reply:
fixed